src/pudl_scrapers/bin/ferc_xbrl.py

```diff
@@ -0,0 +1,131 @@
+"""A command line interface (CLI) to archive data from an RSS feed."""
+import argparse
+import json
+import logging
+import re
+import zipfile
+from pathlib import Path
+
+import feedparser
+import requests
+
+import pudl_scrapers.settings
+from pudl_scrapers.helpers import new_output_dir
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+FERC_RSS_LINK = "https://ecollection.ferc.gov/api/rssfeed"
+
+
+def parse_main():
+    """Process base commands from the CLI."""
+    parser = argparse.ArgumentParser(description="Archive filings from RSS feed")
+    parser.add_argument(
+        "-r", "--rss-path", default=FERC_RSS_LINK, help="Specify path to RSS feed"
```

Comment:
Can you show the default URL in the help message so it's clear what this looks like?